### ISSUE TITLE
Message notifications don't update tab

### ISSUE THREAD

**Issue by Carl2213** [humhub/humhub-modules-mail#190]
Message notifications don't count towards the browser tab's notification count:

![image](https://user-images.githubusercontent.com/69092633/93690590-7148b580-fab0-11ea-9878-289ed12cf1eb.png)

I know you're working on push notifications, but I'm not sure if this is the same or if it needs to be addressed separately.

**Comment by buddh4** [comment]
I think its a bit tricky to sync those values, I think this needs to be solved in the core.

**Comment by buddh4** [comment]
Yes, this currently is only implemented for normal notifications. When adding the mail notification count we need to sync those two values independently. But should be doable, but as I said needs to be done in the core.

**Comment by Carl2213** [comment]
I think mail is the only module that currently needs this, right?

Maybe a possible workaround would be to have the mail module use the default notification system, without displaying those notifications in the front-end (i.e. in the notification dropdown)? This would correctly update the page title, and (probably) correctly implement push notifications for messages, without requiring a lot of work.

**Comment by buddh4** [comment]
Yes, using the notification system would be great and would also solve other issues, but this also would require changes in the core since such a seperation in the frontend is currently not possible.

**Comment by buddh4** [comment]
I thought of something like the `stream_channel` for content to seperate notifications in the frontend. Notifications with `channel_id = default` will affect the normal notifications and notifications with `channel_id = mail_message` will affect the mail module dropdown.
